### ISSUE TITLE
ELS, Control Panel how to change start setting,  starting from metric and not Imperial 

### ISSUE THREAD

**Issue by andrea271068** [clough42/electronic-leadscrew#203]
Good day, James.

Please are you able to assist me.

In the control panel, when I switch on, I would like to start from metric and not Imperial, but because I don't have experience and competence in programming I'm scared to corrupt the code. 

Surfing on, I found that maybe this option is placed in: userinterface.cpp

There is this string: this->metric = false; //start out with imperial

I don't know how to change this input, can you help me, please

**Comment by jimhinze** [comment]
I haven't compiled the code yet, but changing that to this->metric = true;  will default it to metric from the start. It's not set to an initial value in the header, so this is the first place the value is initialized. I'd also change the comment to //start out with metric  so you know what you did.

It won't corrupt the code, it's a one line change.. so if I'm wrong, you can just change back and re-compile, or wait till someone corrects me :)

**Comment by andrea271068** [comment]
this in line 79? I should type: this->metric = true; start out with metric  
instead of:  this->metric = false; start out with imperial

**Comment by jimhinze** [comment]
yes, based on what I see and my understanding with of the code. Bear in mind, I've not compiled it yet (I'm one of the ones waiting on TI boards to be available). So, forgive me if I'm wrong.. someone will correct me :)

**Comment by andrea271068** [comment]
Thanks, let’s try

Andrea Pisani
Overseas markets export manager
Mob: +39 335 70 98 102 - Ph: +39 0375 783687
Mail: ***@***.******@***.***>

Composad srl – Italy

**Comment by jimhinze** [comment]
Andrea,

did it work?

> Thanks, let’s try Andrea Pisani Overseas markets export manager Mob: +39 335 70 98 102 - Ph: +39 0375 783687 Mail: ***@***.******@***.***> Composad srl – Italy

**Comment by andrea271068** [comment]
Yes, many thanks, was very easy 🙂✋
